packages/affiliates/libs/bigquery.js

```diff
@@ -8,7 +8,21 @@ module.exports = ({ client } = {}) => {
 
   // types of queries we are interested in
   const queries = {
-    // logs aka events by contract
+    // this is an expensive call (50gb at time of writing), but is necessary to get all logs without knowing
+    // the deployment time of a contract.
+    allLogsByContract(contract, selection = ["*"]) {
```

Comment:
added another bigquery function to get logs without a date range, vs logsByContract which you specify start and end time